### PR TITLE
Layout page/examples

### LINKED ISSUE #440: Layout Page Convert examples to usage examples
For each example on the Layout page:

- Use <Example /> component
  - Ensure referenced demo code is clean
  - Add related resource links as appropriate:
    - Designer
    - Figma
    - Storybook

### PR DESCRIPTION
**Pull request by britt6612**
Related Issue: #440 

Closes #440 

Work Included: 
* Implemented the controls page examples to use the usage examples
* Each example will now have the more to view the code and other props

### CODE REVIEW

File: aries-site/src/examples/components/layouts/LayerExample.js [hunk 1]
```diff
@@ -1,15 +1,14 @@
...
-import { UsageExample } from '../../../layouts';
 
 export const LayerExample = () => {
   const [open, setOpen] = useState(false);
   const size = useContext(ResponsiveContext);
   const onOpen = () => setOpen(true);
   const onClose = () => setOpen(undefined);
   return (
-    <UsageExample>
+    <Box>
```
Comment (taysea) [inline aries-site/src/examples/components/layouts/LayerExample.js]: Could this outer box just be a <> and then the closing would be </> ?

Comment (halocline) [reply aries-site/src/examples/components/layouts/LayerExample.js]: Since this is going to be an example demonstrating best practices, I think it is a good idea to implement @taysea 's suggestion.

**Comment by britt6612** [comment]
> Looking good. Two small changes for polish:
> 
> 1. [#460 (comment)](https://github.com/hpe-design/design-system/pull/460#discussion_r387275597)
> 2. The padding on the FooterExample is inconsistent (larger) with the rest of the examples on this page. Can we update this example for consistency?
> 
> Screen Shot 2020-03-03 at 1 55 53 PM

I agree the padding is off the initial code was to match the padding of the aries-core nav

```

      pad={{
        vertical: size !== 'small' ? 'small' : 'small',
        // Match horizontal padding of aries-core Nav
        horizontal: size !== 'small' ? 'medium' : 'large',
      }}

```

so not sure if there was a discussion as to why this had to match ?

**Comment by britt6612** [comment]
Found another issue

Screen Shot 2020-03-03 at 2 32 35 PM

File: aries-site/src/examples/components/layouts/FooterExample.js [hunk 2]
```diff
@@ -8,11 +8,6 @@ export const FooterExample = () => {
     <Footer 
       background="background-front"
       direction={size !== 'small' ? 'row' : 'column-reverse'}
-      pad={{
-        vertical: size !== 'small' ? 'small' : 'large',
-        // Match horizontal padding of aries-core Nav
-        horizontal: size !== 'small' ? 'medium' : 'large',
-      }}
       align={size !== 'small' ? 'center' : undefined}
```
Comment (taysea) [inline aries-site/src/examples/components/layouts/FooterExample.js]: Trying to comment something farther down this file, but the styling of the Footer Links doesn't match the most recent Footer styles (it's still using the bolded on hover).

File: aries-site/src/examples/components/layouts/HeaderExample.js [hunk 3]
```diff
@@ -8,41 +8,40 @@ export const HeaderExample = () => {
...
-      >
-        {size !== 'small' || focused ? (
-          <TextInput placeholder="Search HPE Design System" plain />
-        ) : (
-          undefined
-        )}
+      <Header fill="horizontal" pad="none" background='background-front'>
+        <AppIdentity
+          brand="hpe"
+          logoOnly={size === 'small'}
```
Comment (taysea) [inline aries-site/src/examples/components/layouts/HeaderExample.js]: I think we'd still want the HPE Design System text to appear unless the TextInput is focused, this can be in a separate PR though if you'd prefer.
Screen Shot 2020-03-06 at 10 18 50 AM

Comment (britt6612) [reply aries-site/src/examples/components/layouts/HeaderExample.js]: yes I agree I think since Ill have to look at the logic of how that is hidden in the search component so I think this will be better to put in another PR I did add this to issue #445

File: aries-site/src/examples/components/layouts/FooterExample.js (hunk 2 above)
Comment (britt6612) [reply aries-site/src/examples/components/layouts/FooterExample.js]: oh yes I changed that in the footer PR I just wasn't up to date with master should be good now!
